src/main/java/com/minecolonies/coremod/util/RecipeHandler.java

```diff
@@ -135,6 +135,8 @@ public static void init(final boolean enableInDevelopmentFeatures, final boolean
                 new ItemStack(ModBlocks.blockShingleDarkOak, EIGHT_BLOCKS, 0), "B  ", "SB ", "PSB", 'B', Items.BRICK, 'P',
                 new ItemStack(Blocks.PLANKS, 1, 5), 'S', WOODEN_STICK));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 5 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)